src/app/shared/helpers/arrays.ts

```diff
@@ -0,0 +1,16 @@
+function getRandom(min, max): number {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
+export function shuffleArray<Type>(arr: Array<Type>): Array<Type>  {
```

Comment:
Is this any particular algorithm?

It looks like knuth shuffle, but there are some differences in the code https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array although I'm not sure if they are significant.

Reply:
It is knuth shuffle. Do you want me to specify it in the comment above the function?

Reply:
No need, you're good to go :)